src/config/configureStore.js

```diff
@@ -15,12 +16,13 @@ const auth = useAuthServer ? authReducer : null;
 export const store = configureStore({
   reducer: {
     auth,
+    modal: modalReducer,
     [postApi.reducerPath]: postApi.reducer,
     [dinersApi.reducerPath]: dinersApi.reducer,
     [reviewApi.reducerPath]: reviewApi.reducer,
   },
   middleware: (getDefaultMiddleware) =>
-    getDefaultMiddleware()
+    getDefaultMiddleware({ serializableCheck: false })
```

Comment:
What is this for?

Reply:
This is so you can pass a component to the redux store